@xen-orchestra/backups/writers/DeltaBackupWriter.js

```diff
@@ -200,20 +200,27 @@ exports.DeltaBackupWriter = class DeltaBackupWriter extends MixinBackupWriter(Ab
           const vdiRef = vm.$xapi.getObject(vdi.uuid).$ref
 
           let nbdClient
-          if (!this._backup.config.useNbd) {
+          if (this._backup.config.useNbd) {
```

Comment:
node the boolean change

Reply:
This is a fix and should be in a separate commit.

Let me know if you want me to rebase the history.